common/src/main/resources/data/adorn/recipes/brown_sofa.json

```
@@ -11,7 +11,7 @@
       "item": "minecraft:stick"
     },
     "W": {
-      "item": "minecraft:brown_wool"
+      "items": ["minecraft:brown_wool"]
```

```suggestion
      "item": "minecraft:brown_wool"
```
This is not an advancement.